src/core/lib/debug/trace.h

```diff
@@ -53,7 +53,8 @@ void grpc_tracer_enable_flag(grpc_core::TraceFlag* flag);
 class TraceFlag {
  public:
   TraceFlag(bool default_enabled, const char* name);
-  ~TraceFlag() {}
+  // This needs to be trivially destructible as it is used as global variable.
+  ~TraceFlag() = default;
```

Comment:
Let me suggest that there's an explicit `static_assert(std::is_trivially_destructible` to make sure that this never arises in the future and that we don't make this a class with virtual destructor etc.

Reply:
Good point. Done.